### PR TITLE
pipeline ground truth

### PR DESCRIPTION
**Pull request by TashaSkyUp**
# This is a phase for the training data pipeline which assists in entering the ground truth for lines of writing extracted from the story photograph by the previous phases. There is also a couple of bug fixes which where completed while testing and creating this. see commit messages.

## Type of change

https://user-images.githubusercontent.com/15312630/136877841-c27c3cf4-e30c-4b4e-a48a-6a4b98eea26b.mp4

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by Fellylove — APPROVED** [review]
Great work, Tasha!

We need more data for the tesseract model and this application will get us there. I have viewed all 7 files and they're good to go. There are just a few docstrings that need to be added to some classes.

**Review by cryptobellum — APPROVED** [review]
As usual, you've gone above and beyond. Great work!

**Review by techthumb1 — COMMENTED** [review]
Great job Tasha! Very clean and easy to follow the code. Like the addition of the scrollbar, and   awesome documentation which also is appreciated.

File: data_management/story_photo_ground_truth.py [hunk 1]
```diff
@@ -0,0 +1,239 @@
+import os
+import tkinter as tk
+from enum import IntFlag, auto
+
+import cv2
+import numpy as np
```
Comment (singparvi) [inline data_management/story_photo_ground_truth.py:6]: I may be wrong but I guess the following imports can be done away with:-
```python 
from enum import IntFlag, auto

import cv2
import numpy as np
from data_management.phase_tkinter_class import np_photo_image
```

**Comment by singparvi** [comment]
Nice work @TashaSkyUp. This in itself will be a very important tool to validate existing data on the fly. Much better than what I planned for but could not complete. I planned to do it with two Tkinter frames on the left showing the text and on the right the image itself. With your utility a number of snippets can be reviewed at the same time.

**Review by abdishifa234 — APPROVED** [review]
This is nice work but I couldn't understand the whole codes,I wish some more  description should be add the repos. Overall great job!

**Pull request merged**